### PR TITLE
Add pulumi image

### PR DESCRIPTION
**Pull request by jdolitsky**
Fixes:

Related: 

### Pre-review Checklist

- [ ] **IMPORTANT: 'image-request' tag has been applied if this PR is adding any images, including new versions or variants**

#### For new image PRs only

If you have an apko.yaml file in this PR you need to follow this checklist, otherwise feel free to remove.
- [x] Image is marked experimental or stable as appropriate
- [x] The version included is the latest GA version of the software
- [x] The latest tag points to the newest stable version
- [ ] There is a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base')
- [x] The image runs as `nonroot` and GID/UID are set to 65532
  - [ ] Alternatively the username and GID/UID may be a commonly used one from the ecosystem e.g: postgres
  - [ ] See above for exceptions to nonroot rule
- [x] ENTRYPOINT
  - [ ] For applications/servers/utilities call main program with no arguments e.g. [redis-server]
  - [ ] For base images leave empty
  - [ ] For dev variants set to entrypoint script that falls back to system
- [x] CMD:
  - [ ] For server applications give arguments to start in daemon mode (may be empty)
  - [ ] For utilities/tooling bring up help e.g. `–help`
  - [ ] For base images with a shell, call it e.g. [/bin/sh]
- [ ] Consider where and how the image deviates from popular alternatives. Is there a good reason and is it documented?
- [x] Add annotations e.g:
```
annotations:
  "org.opencontainers.image.authors": "Chainguard Team https://www.chainguard.dev/"
  "org.opencontainers.image.url": https://edu.chainguard.dev/chainguard/chainguard-images/reference/busybox/ # use the academy site here
  "org.opencontainers.image.source": https://github.com/chainguard-images/images/tree/main/images/bazel # use github here
```
- [ ] Check if environment variables are needed e.g. to set data locations
- [ ] Ensure the image responds to SIGTERM
  - [ ] `docker kill $(docker run -d --rm cgr.dev/chainguard/nginx)`
- [x] Documentation. Let's make this excellent. Include usage example.
- [x] Error logs write to stderr and normal logs to stdout. DO NOT write to file.
- [x] Include tests, at the very least a basic smoke test.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,7 @@
...
 | [prometheus-node-exporter](./images/prometheus-node-exporter) | `cgr.dev/chainguard/prometheus-node-exporter` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/prometheus-node-exporter.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/prometheus-node-exporter:latest) |
 | [prometheus-postgres-exporter](./images/prometheus-postgres-exporter) | `cgr.dev/chainguard/prometheus-postgres-exporter` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/prometheus-postgres-exporter.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/prometheus-postgres-exporter:latest) |
+| [pulumi](./images/pulumi) | `cgr.dev/chainguard/pulumi` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/pulumi.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/pulumi:latest) |
```
Comment (amdawson) [inline README.md:78]: does it need to be experimental? do we know of issues with the image or do expect breaking changes?

Comment (jdolitsky) [reply README.md:78]: let me make sure it works in all the supported languages :)

Comment (jdolitsky) [reply README.md]: switched to stable

File: images/pulumi/README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+mkdir .pulumi
+docker run --rm --network kind \
+    -w "/work/examples/smoketest-${TEST_LANG}" \
+    -v "${PWD}:/work" \
+    -e PULUMI_HOME=/work/.pulumi \
+    cgr.dev/chainguard/pulumi:latest \
+    login file://.
+```
+
+Decide a unique stakc name:
```
Comment (patflynn) [inline images/pulumi/README.md:77]:
```suggestion
Decide a unique stack name:
```

File: images/pulumi/configs/latest.apko.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    - pulumi-language-nodejs
+    - nodejs
+    - nghttp2
+
+    # for Pulumi Java support
+    - pulumi-language-java
+    - glibc-locale-en
+    - openjdk-17-jre
+    - libstdc++
+    - maven
```
Comment (patflynn) [inline images/pulumi/configs/latest.apko.yaml:41]: I'm surprised it needs maven but only the jre version of java. Is it doing builds?  If so it should probalby include the full jdk.

File: images/pulumi/configs/latest.apko.yaml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+    # for Pulumi Node.js support
+    - pulumi-language-nodejs
+    - nodejs
+    - nghttp2
+
+    # for Pulumi Java support
+    - pulumi-language-java
+    - glibc-locale-en
+    - openjdk-17-jre
```
Comment (jdolitsky) [inline images/pulumi/configs/latest.apko.yaml:39]:
```suggestion
    - openjdk-17-jre
    - openjdk-17
```

**Pull request merged**